ui/app/templates/partials/tools/unwrap.hbs

```diff
@@ -8,19 +8,40 @@
 
 {{#if unwrap_data}}
   <div class="box is-sideless is-fullwidth is-marginless">
-    <div class="field">
-      <label class="is-label">
-        Unwrapped data
-      </label>
-      <div class="control">
-        {{json-editor
-          value=(stringify unwrap_data)
-          options=(hash
-            readOnly=true
-          )
-        }}
+    <nav class="tabs">
+      <ul>
+        <li class="{{if (eq unwrapActiveTab "data") "is-active"}}">
+          <a href="#" {{action "updateUnwrapTab" "data"}} class="link link-plain has-text-weight-semibold {{if (eq unwrapActiveTab "data") "is-active"}}">Data</a>
+        </li>
+        <li class="{{if (eq unwrapActiveTab "details") "is-active"}}">
+          <a href="#" {{action "updateUnwrapTab" "details"}} class="link link-plain has-text-weight-semibold {{if (eq unwrapActiveTab "details") "is-active"}}">Wrap Details</a>
+        </li>
+      </ul>
+    </nav>
+    {{#if (eq unwrapActiveTab "data")}}
+      <div class="field">
+        <div class="control">
+          {{json-editor
+            value=(stringify unwrap_data)
+            options=(hash
+              readOnly=true
+            )
+          }}
+        </div>
       </div>
-    </div>
+    {{else}}
+      <div class="field box is-fullwidth is-shadowless is-paddingless is-marginless">
+        {{#each-in details as |key detail|}}
```

Comment:
So why details and unwrap_data? I think we want to show all of the unwrap_data here too, just in a bit nicer format.